WordPressKitTests/Utilities/URLSessionHelperTests.swift

```diff
@@ -227,6 +236,92 @@ class URLSessionHelperTests: XCTestCase {
         let expectedBody = "--\(boundary)\r\nContent-Disposition: form-data; name=\"name\"\r\n\r\nvalue\r\n--\(boundary)--\r\n"
         XCTAssertEqual(String(data: requestBody, encoding: .utf8), expectedBody)
     }
+
+    func testGetLargeData() async throws {
+        let file = try self.createLargeFile(megaBytes: 100)
+        defer {
+            try? FileManager.default.removeItem(at: file)
+        }
+
+        stub(condition: isPath("/hello")) { _ in
+            HTTPStubsResponse(fileURL: file, statusCode: 200, headers: nil)
+        }
+
+        let builder = HTTPRequestBuilder(url: URL(string: "https://wordpress.org/hello")!)
+        let response = try await session.perform(request: builder, errorType: TestError.self).get()
+
+        try XCTAssertEqual(
+            sha256(XCTUnwrap(InputStream(url: file))),
+            sha256(InputStream(data: response.body))
+        )
+    }
```

Comment:
This is now the slowest test in the suite, multiplied by the number of subclasses. [Source](https://buildkite.com/organizations/automattic/analytics/suites/wordpress-ios-kit/runs/3b366dac-1daa-8c9d-87c0-d0d8c6d43ac6).

![image](https://github.com/wordpress-mobile/WordPressKit-iOS/assets/1218433/61cbdbbd-0986-4d52-9deb-33cef7ca5af2)

I suppose part of that is generating the large file at runtime. I wonder what the implications would be of tracking the file in the test suite for repo size and test duration. Not a big deal at this point, anyways.